src/components/OutlinedButton.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <a
+    :disabled="disabled"
+    class="button-link"
+    :href="href"
+    :target="target"
+    :title="title"
```

Comment:
It may be written more shortly, like:
```suggestion
    class="button-link"
    v-bind="{ disabled, href, target, title }"
```